setup.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name='cmsplugin-articles-ai',
+    version='0.0.1',
+    author='Anders Innovations',
+    author_email='info@anders.fi',
+    packages=[
```

Comment:
Use `find_packages`. Otherwise you'll inevitably forget to add a package here...